clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClient.java

```diff
@@ -1222,6 +1222,8 @@ static ZeebeClientCloudBuilderStep1 newCloudClientBuilder() {
    *  .newCreateMappingCommand()
    *  .claimName(claimName)
    *  .claimValue(claimValue)
+   *  .name(name)
+   *  .operator(CONTAINS)
```

Comment:
🔧 as discussed, `operator` is no longer needed in the new architecture because we use a key of `claim_name`+`claim_value` to perform a search with in Zeebe. Could we update this PR to only add the `name` field.